specreduce/tests/test_tracing.py

```diff
@@ -183,7 +183,41 @@ def test_window_fit_trace(self):
 
     @pytest.mark.filterwarnings("ignore:The fit may be unsuccessful")
     @pytest.mark.filterwarnings("ignore:Model is linear in parameters")
-    def test_fit_trace_all_nan_columns(self):
+    @pytest.mark.filterwarnings("ignore:All pixels in bins")
+    def test_fit_trace_all_nan_cols(self):
+        pass
```

Comment:
does this just skip the test?
```suggestion
```

Reply:
oh whoops, i dont know how that snuck in there